Makefile

```
@@ -0,0 +1,130 @@
+.PHONY: help simulator-up simulator-down simulator-list simulator-status test-ios test test-all install setup clean precommit-install precommit
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)